### ISSUE TITLE
'vue/mustache-interpolation-spacing' is broken in eslint-plugin-vue 5.2.3

### ISSUE THREAD

**Issue by aljones15** [vuejs/eslint-plugin-vue#942]
**Tell us about your environment**	
* **ESLint version:** ^6.1.0

* **eslint-plugin-vue version:** ^5.2.3

* **Node version:**	11.13

**Please show your full configuration:**

```json
module.exports = {
  env: {
    browser: true
  },
  extends: ['plugin:vue/recommended'],
  rules: {
    'vue/mustache-interpolation-spacing': 'never',
    'vue/html-closing-bracket-newline': ['error', {
      singleline: 'never',
      multiline: 'never'
    }]
  }
};
```

**What did you do?**	

```vue
any vue file works
```

**What did you expect to happen?**
there would be no error

**What actually happened?**

Error: .eslintrc.js » eslint-config-digitalbazaar/vue:
  2         Configuration for rule "vue/mustache-interpolation-spacing" is invalid:
  3         Severity should be one of the following: 0 = off, 1 = warn, 2 = error (you passe    d '"never"').

"never" is still valid according to your docs:

https://eslint.vuejs.org/rules/mustache-interpolation-spacing.html

and your own rule set uses warn:

https://github.com/vuejs/eslint-plugin-vue/blob/master/lib/configs/strongly-recommended.js

**Comment by ota-meshi** [comment]
Hello!
You can change the settings as follows:

```diff
- 'vue/mustache-interpolation-spacing': 'never',
+ 'vue/mustache-interpolation-spacing': ['error', 'never'],
```

https://eslint.vuejs.org/rules/mustache-interpolation-spacing.html#options

**Comment by aljones15** [comment]
thanks man that fixed it!
